README.md

```diff
@@ -40,8 +40,9 @@ Be sure to remove or modify the `username` attribute.
 
 ```sh
 $ bundle exec rake db:create \
->  && bundle exec rake db:schema:load \
->  && bundle exec rake db:seed
+  && bundle exec rake db:migrate \
```

Comment:
Ah good catch. `db:create` and `db:migrate` need to be done in a separate context. I've bumped into that before. It's no issue just to leave it as-is :+1: